### ISSUE TITLE
Reports of a neutral item removed from ffa and duos appearing in enfos

### ISSUE THREAD

**Issue by Snoresville** [arcadia-redux/custom_hero_clash_issues#1665]
![image](https://user-images.githubusercontent.com/19840964/135707837-ef2951a1-0f65-472f-994b-d6fcd913e590.png)
![image](https://user-images.githubusercontent.com/19840964/135707847-c38e78a4-34c2-47f5-9bbe-9a6ab9cad964.png)
![image](https://user-images.githubusercontent.com/19840964/135707850-314c7072-fa6d-4d38-bc63-29cc6440d12e.png)

If this is intentional, close this.

**Comment by fcalife** [comment]
Its intentional, but could be changed if we wish. Its a treasury drop.

**Comment by Draze22** [comment]
Isn't Turbulent Sturmaz a tier 5 item?

**Comment by Snoresville** [comment]
yes

**Comment by fcalife** [comment]
yes, some t5 items appear on enfos treasury if you wait long enough (they're not obtainable otherwise, currently)
